packages/flutter/lib/src/material/app.dart

```diff
@@ -166,6 +169,17 @@ class MaterialApp extends StatefulWidget {
   /// The colors to use for the application's widgets.
   final ThemeData theme;
 
+  /// The colors to use for the application's widgets when the host platform has
+  /// specifically requested a dark UI.
+  ///
+  /// For example, starting with Android Pie, turning on battery saver causes
+  /// apps to go into "dark mode".
+  ///
```

Comment:
Directly address how this is different from (Cupertino)ThemeData.brightness

Reply:
@xster can you describe how the 2 would be confused?  I'm not sure how to explain the difference because I'm not sure how they're similar to begin with....

Reply:
One is an incoming signal from the platform and one is an outgoing effect that describes what descendent widgets look like.